templates/consul.service.j2

```diff
@@ -0,0 +1,17 @@
+[Unit]
+Description=Consul Agent
+Requires=network-online.target
+After=network-online.target
+
+[Service]
+User=consul
```

Comment:
If we run Consul as root, we could use its ability to run commands on nodes, but we could achieve something similar by giving sudo privileges to the consul user.  Overall it seems better to let Consul run as a non-root user.

Reply:
Leaving this as non-root then :+1: